src/main/java/nl/hsac/fitnesse/fixture/slim/HttpTest.java

```diff
@@ -24,7 +24,9 @@
  * Fixture to make HTTP requests using Slim scripts and/or scenarios.
  */
 public class HttpTest extends SlimFixtureWithMap {
-    /** Default content type for posts and puts. */
+    /**
```

Comment:
reformatting JavaDoc seems to make this PR a bit hard to read.
I'm not really bothered about fixing these, but if you are please make a PR for the entire project.

Reply:
Yeah I have a tendency top press control alt L a lot... I'll revert these.